### PR TITLE
Issue/1630 – JavaScript debug info API

### PR DESCRIPTION
**Pull request by ramiabraham**
Issue: #1630

### CODE REVIEW

File: assets/js/tracking.js [hunk 1]
```diff
@@ -115,4 +132,122 @@ jQuery(document).ready( function($) {
...
+			 */
+			vars : {
+				ajax_url        : affwp_scripts.ajaxurl,
+				ref_cookie      : ref_cookie,
+				visit_cookie    : visit_cookie,
+				credit_last     : AFFWP.referral_credit_last,
+				campaign_cookie : campaign_cookie,
+				ref             : AFFWP.referral_var,
+				campaign        : affwp_get_query_vars()['campaign'],
+			    currency        : affwp_debug_vars.currency,
```
Comment (pippinsplugins) [inline assets/js/tracking.js]: Indention needs fixed here

Comment (DrewAPicture) [inline assets/js/tracking.js]: This is fixed.

**Comment by ramiabraham** [comment]
@AffiliateWP/core I'd like to pause here to request testing on this pretty please. Despite considerable testing, I'm getting variable behavior from browser caches and would love to see how consistent this is on other machines:

- Ensure AffWP debug is active in Settings -> Misc
- Load the site with one or more affwp query vars (ref, visit id, and campaign are all logged with the debugger)
- Check the browser console
    - What you should see: two tables of debug data; one for primary debug data such as the `ref`, one listing any active affwp integrations
   - Showing instead: empty `console.table` outputs

Any testing/thoughts appreciated.

**Comment by pippinsplugins** [comment]
@ramiabraham I get two JS errors:

- Uncaught SyntaxError: Unexpected token ; - line 256
- Uncaught ReferenceError: AFFWP is not defined - tracking line 7

**Pull request merged**